io.catenax.request_for_quotation/1.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,584 @@
+#######################################################################
+ # Copyright (c) 2023 Fraunhofer Institute of Optronics, System Technology and Image Exploitation (IOSB)
+ # Copyright (c) 2023 Contributors to the Eclipse Foundation
+ #
+ # See the NOTICE file(s) distributed with this work for additional
+ # information regarding copyright ownership.
+ #
+ # This work is made available under the terms of the
+ # Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+ # which is available at
+ # https://creativecommons.org/licenses/by/4.0/legalcode.
+ #
+ # SPDX-License-Identifier: CC-BY-4.0
+ #######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.request_for_quotation:1.0.0#> .
+
+:RequestForQuotation a samm:Aspect ;
+   samm:preferredName "Request For Quotation"@en ;
+   samm:description "The request for quotation represents the data structure to configure a manufacturing request and send this to a supplier receive a quotation"@en ;
+   samm:properties ( :senderBpn :recipientBpn :context :messageId :sentDateTime :version :requestForQuotationSource :requestForQuotationId :requestForQuotationName :requestForQuotationDateTime :cadFileType :cadFile :partMaterials :billOfMaterial :manufacturingDomains :manufacturingMethods :postProcesses :billOfProcess :generalTolerance :additionalQualityRequirements :testPartRequired :quantity :lastDeliveryDate :firstDeliveryDate :additionalComments :additionalFiles :senderName :senderAdress :senderCompanyName :senderEMail :senderPhoneNumber :senderAccountAdress :senderDeliveryAdress :deliveryRequirements ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:senderBpn a samm:Property ;
+   samm:preferredName "sender BPN"@en ;
+   samm:description "CX-Header: BPN identification of sender partner"@en ;
+   samm:characteristic :BpnTrait ;
+   samm:exampleValue "BPNL50096894aNXY" .
+
+:recipientBpn a samm:Property ;
+   samm:preferredName "recipient BPN"@en ;
+   samm:description "CX-Header: BPN identification of recipient partner"@en ;
+   samm:characteristic :BpnTrait ;
```

Comment:
Here also please check if you can make use of shared aspect model https://github.com/eclipse-tractusx/sldt-semantic-models/blob/a07bdd5b1515767d4f8d07deb47cf2a4aa728bef/io.catenax.shared.business_partner_number/1.0.0/BusinessPartnerNumber.ttl